project-base/storefront/hooks/user/useCurrentUserContactInformation.ts

```diff
@@ -21,7 +21,7 @@ const mergeContactInformation = (
         const filteredProperty = filteredContactInformationFromStore[key as keyof ContactInformation];
 
         const isEmptyString = typeof filteredProperty === 'string' && filteredProperty.length === 0;
-        if (isEmptyString && key in contactInformationFromApi) {
+        if ((isEmptyString || filteredProperty === null) && key in contactInformationFromApi) {
```

Comment:
Můžeme odebrat check pro `null`?

```suggestion
        if ((isEmptyString || !filteredProperty) && key in contactInformationFromApi) {
```

Reply:
I don't think it makes sense. We do not want to filter out all falsy values, just empty strings and nulls.

Reply:
Ah, ok. It was not obvious from code here. But I've checked and I see also `undefined` as possible value, don't you want to include it as well?

Reply:
I did, thanks for the suggestion